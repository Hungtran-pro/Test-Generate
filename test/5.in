40
78
63
105
116
87
106
61
102
105
62
116
116
65
88
76
101
76
81
120
102
113
67
85
108
106
105
115
103
117
117
120
60
96
98
63
105
95
68
118
105
