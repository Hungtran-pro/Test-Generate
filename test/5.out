116509860
881779447
676689719
670858904
241802082
56249491
312450027
635707616
676689719
217689433
670858904
670858904
562992784
142382092
376671503
351513474
376671503
459405516
951234175
635707616
642654783
718461253
100543565
419333973
56249491
676689719
721568908
981043477
273207878
273207878
951234175
266177107
103131504
892187321
881779447
676689719
398885785
599807473
904799209
676689719
